src/components/SearchBar/index.js

```diff
@@ -1,32 +1,41 @@
 import { useRouter } from 'next/router';
-import SearchBar from "material-ui-search-bar"
 import React from "react";
 
 import styles from './styles.module.css';
 import { sendEvent } from 'hooks/amplitude';
+import SearchIcon from '@material-ui/icons/Search';
 
-const Menu = (props) => {
+const SearchBar = (props) => {
```

Comment:
i think we should add a prop that indicates whether or not the search bar is in the corner or on the homepage, and then we can use that to make some small design changes like the look of the button and also the box-shadow.

something like `isHomepage` and then just pass in true on the homepage

Reply:
Added in the logic. Didn't have specific styling to apply, so just made empty classes for now.